docs/how-to/deploy_cups_sdcore.md

```diff
@@ -0,0 +1,217 @@
+# Deploy SD-Core with CUPS
+
+This guide covers how to install a multi-node sdcore with separation of user and control plane traffic.
+
+### Requirements
+
+You will need two kubernetes clusters as follows.  In this how-to, we cover installation and configuration of MicroK8s, however, any compatible Kubernetes cluster can be used.
+- `SD-Core Control Plane`, where the full set of control plane functions will be placed (AMF, SMF, etc)
+- `SD-Core User Plane`, where the User Plane Function (UPF) will be placed
+
+A separate machine is required for the Juju controller.  While it is possible to install Juju on the same host as the SD-Core Control Plane, it is not recommended to do so.  [`TODO:` note on metallb for juju controller]
+
+In the `SD-Core User Plane` cluster, the following subnets must be present:
+- `access`, where the UPF listens for incoming sessions from the gNB
+- `core`, where the UPF send out traffic from the gNB towards the internet
+- `management`, for Juju and control plane traffic.  Must also have access to the internet for downloading container images, charms, or other updates as needed
+
+[`TODO:` Add note about port security somewhere for the networks]
+
+##### Sample Values
+
+For the purpose of this guide, the following values will be used.
+
+###### Subnets
+
+| Name | VLAN | Subnet | Gateway IP |
+| ---- | ---- | ------ | ---------- |
+| management | 1201 | 10.201.0.0/24 | 10.201.0.1 |
+| access     | 1202 | 10.202.0.0/24 | 10.202.0.1 |
+| core       | 1203 | 10.203.0.0/24 | 10.203.0.1 |
+
+###### IP Addresses
+
+The only DNS entry that is required is for the AMF so that the gNB can be told where to find it [`TODO:` link to gnb interface reference]
+
+| Name | IP Address | Purpose |
+| ---- | ---------- | ------- |
+| juju-controller.k8s.priv | 10.201.0.10 | Management address for Juju machine
+| control-plane.k8s.priv | 10.201.0.11 | Management address for control plane cluster machine | 
+| user-plane.k8s.priv | 10.201.0.12 | Management address for user plane cluster machine | 
+| control-plane.juju.priv | 10.201.0.101 | Juju controller load balancer port |
+| upf.core.priv | 10.201.0.151 | IP address for UPF to bind to |
+| amf.core.priv | 10.201.0.201 | Externally reachable control plane endpoint for the AMF |
+
+
+`````{tab-set}
```

Comment:
I'm not sure about the use of tab sets here, in the reference we use them as options (one **or** the other) and here we use them as a list of things to do (one **and** the other)

Reply:
can we come up with some guidelines then?